### PR TITLE
test(echo): Mvp test suite

### PR DESCRIPTION
**Pull request by djabarovgeorge**
### What changed? Why was the change needed?
* Define a basic MVP journey for SDK 
* Add tests for basic MVP journey

### Screenshots

<details>
<summary><strong>Expand for optional sections</strong></summary>

### Special notes for your reviewer

</details>

### CODE REVIEW

File: packages/sdk/src/schemas/providers/chat/slack.ts
Comment (djabarovgeorge) [inline packages/sdk/src/schemas/providers/chat/slack.ts:1]: Git behaved super strange and i lost track of what is the last version.
@scopsy i think you added/fixed the last slack scheme, those this seem right to you? please be aware that in my PR should not be any changes regarding Slack.

File: packages/sdk/src/types/step.types.ts [hunk 1]
```diff
@@ -12,7 +12,6 @@ import { MaybePromise } from './util.types';
 export type ActionStepOptions = {
   skip?: Skip;
   inputSchema?: Schema;
-  outputSchema?: Schema;
```
Comment (djabarovgeorge) [inline packages/sdk/src/types/step.types.ts:15]: not sure why we had it next, [REF](https://github.com/novuhq/echo/pull/1#discussion_r1590334544)

File: packages/sdk/src/schemas/providers/chat/slack.ts
Comment (ainouzgali) [reply packages/sdk/src/schemas/providers/chat/slack.ts:1]: Try to update the branch and then changing the destination branch and then changing back to next. Sometimes it fixes the weird unrelated files

**Review by rifont — APPROVED** [review]
Awesome work!

**Pull request merged**